client/src/components/Sidebar/Contacts/Contacts.js

```diff
@@ -7,43 +8,66 @@ import Friends from './Friends';
 import Requests from './Requests';
 import Pending from './Pending';
 
+const useStyles = makeStyles(theme => ({
+  invitationHeadings: {
+    marginBottom: 0
+  }
+}));
+
 const Contacts = props => {
-  const [display, setDisplay] = React.useState('friends');
+  const classes = useStyles();
+  const [display, setDisplay] = useState('friends');
   const toggleDisplay = (event, setting) => {
     setDisplay(setting);
   }
- const [friends, setFriends] = React.useState([]);
+ const [friends, setFriends] = useState([]);
+ const [pendingInvites, setPendingInvites] = useState([]);
+ const [pendingRequests, setPendingRequests] = useState([]);
+
  const email = localStorage.getItem('email'); 
  const loadFriends = async() => {
    const res = await axios.get(`http://localhost:3001/invitations/user/${email}/contacts`);
    if(res.data.contacts.length !== 0){
-    setFriends([...res.data.contacts])
+    setFriends([...res.data.contacts]);
```

Comment:
the spread operator here is redundant. Since contacts is an array, you can simply do `setFriends(res.data.contacts)`.
Same thing applies to `setPendingRequests` and `setPendingInvites` below.